engine/src/flutter/shell/common/switches.h

```diff
@@ -300,6 +300,9 @@ DEF_SWITCH(DisableMergedPlatformUIThread,
 DEF_SWITCH(EnableAndroidSurfaceControl,
            "enable-surface-control",
            "Enable the SurfaceControl backed swapchain when supported.")
+DEF_SWITCH(ImpellerShaderMode,
+           "impeller-shader-mode",
+           "Controls shader initialization strategy for the Impeller backend.")
```

Comment:
Maybe document choices?  ["lazy", undefined]

Reply:
Changed to bool flag